### DISCUSSION TITLE
Astro and Tauri

### DISCUSSION THREAD

**Discussion by pierredewilde** [withastro/roadmap#354] [Proposal]
Both Next.js and SvelteKit are officially supported by Tauri:

https://tauri.app/v1/guides/getting-started/setup/

Astro should be too.

See https://github.com/tauri-apps/tauri/discussions/4349

**Comment by mikayelgr** [comment]
I don't think that this is related to RFCs specifically, but you can actually use it with Tauri and seems like somebody has already given an answer to that question https://github.com/tauri-apps/tauri/discussions/4349#discussioncomment-2966362

**Comment by florian-lefebvre** [comment]
Closing because this is outdated
